test/backend/matmul.in.cpp

```diff
@@ -0,0 +1,241 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
```

Comment:
Could we please add lit tests for the lowering of all the MKLDNN ops this PR is introducing to make sure that the the lowering is the expected, that we are passing the right arguments to the dispatcher, etc.?